spec/support/faux_writer.rb

```diff
@@ -20,6 +35,7 @@ def write(trace)
     @mutex.synchronize do
       super(trace) if @options[:call_original]
       @traces << trace
+      @options[:transport].send_traces(trace)
```

Comment:
@wconti27 I wonder if we need this line here. 

The call to `super(trace`) will call the transport to send the traces. It depends on which transport we are using. Since it could be the case that `call_original` is set to true, there might be cases where we are sending the traces twice.